### PR TITLE
Add test for Definition with exeption InvalidArgument

### PR DESCRIPTION
**Pull request by janczer**
Add test for Definition with exeption InvalidArgument

| Q | A |
| --- | --- |
| Branch? | "master" for new features / 2.7, 2.8, 3.0 or 3.1 for fixes |
| Bug fix? | no |
| New feature? | no |
| Deprecations? | no |
| Tests pass? | yes |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Component/Workflow/Tests/DefinitionTest.php [hunk 1]
```diff
@@ -17,6 +17,15 @@ public function testAddPlaces()
         $this->assertEquals('a', $definition->getInitialPlace());
     }
 
+    /**
+     * @expectedException Symfony\Component\Workflow\Exception\InvalidArgumentException
```
Comment (ro0NL) [inline src/Symfony/Component/Workflow/Tests/DefinitionTest.php]: Should be `\Symfony\Component\Workflow\Exception\InvalidArgumentException`

Comment (fabpot) [inline src/Symfony/Component/Workflow/Tests/DefinitionTest.php]: indeed

**Comment by fabpot** [comment]
Thank you @IvanMenshykov.

File: src/Symfony/Component/Workflow/Tests/DefinitionTest.php (hunk 1 above)
Comment (fabpot) [inline src/Symfony/Component/Workflow/Tests/DefinitionTest.php]: 9c74d63

**Pull request closed without merging**